src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -110,10 +134,23 @@ export default App.extend({
     'change:filters': 'restart',
     'change:actionsSortId': 'onChangeStateSort',
     'change:flowsSortId': 'onChangeStateSort',
+    'change:selectedFlows': 'onChangeSelected',
+    'change:selectedActions': 'onChangeSelected',
   },
   onChangeStateSort() {
     this.getChildView('list').setComparator(this.getComparator());
   },
+  onChangeSelected() {
+    const selected = _.keys(this.getState().getSelected(this.collection));
```

Comment:
I'd bet there could be one function that shows/clears depending on `selected.length` that could be called here or in onStart